docs/source/release_notes.rst

```diff
@@ -17,6 +17,7 @@ Release Notes
     * Changes
         * Added `allow_writing_files` as a named argument to CatBoost estimators. :pr:`1202`
         * Added `solver` and `multi_class` as named arguments to LogisticRegressionClassifier :pr:`1202`
+        * Replaced pipeline's `.transform` method with public method to evaluate all the preprocessing steps of a pipeline :pr:`1231`
```

Comment:
Maybe we should add a line to the breaking changes section as well? I agree that changing a private api doesn't normally constitute a breaking change but some users may be relying on `_transform` from old releases?